cmd/config/init/flags.go

```diff
@@ -0,0 +1,47 @@
+package initconfig
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func addFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP(FlagNames.HubRPC, "", defaultHubRPC, "Dymension Hub rpc endpoint")
+	cmd.Flags().StringP(FlagNames.LightNodeEndpoint, "", "", "The data availability light node endpoint. Runs an Arabica Celestia light node if not provided")
```

Comment:
I believe the flag name should indicate DA light node ( we will have in the future also the hub light node)